src/sentry/snuba/events.py

```diff
@@ -566,3 +566,11 @@ class Columns(Enum):
         discover_name="contexts[trace.client_sample_rate]",
         alias="trace.client_sample_rate",
     )
+
+    APP_START_TYPE = Column(
+        group_name=None,
+        event_name=None,
+        transaction_name="app_start_type",
+        discover_name="app_start_type",
+        alias="app.start_type",
```

Comment:
I'm not sure if this is the correct alias. What it is used for?

Reply:
This is the public alias, ie. what customers will query in the UI

Reply:
Thanks, then I will change it.